src/main/webapp/js/finscholar/commonlistview/commonlistview.js

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** @fileoverview The mini controller for scholarship list view. */
+
+goog.module('finscholar.commonlistview');
+
+const {ScholarshipListDataHandler} = goog.require('datahandlers.scholarshiplistdatahandler');
```

Comment:
There's a compile error here that's causing the rest of your branches to break. required namespace 'datahandlers.scholarshiplistdatahandler' is not defined

Reply:
Yeah I created a temporary file which generates data locally. Just added that file.